src/SFA.DAS.AANHub.Application/Employers/Commands/CreateEmployerMemberCommandValidator.cs

```diff
@@ -18,27 +18,22 @@ public class CreateEmployerMemberCommandValidator : AbstractValidator<CreateEmpl
 
             Include(new CreateMemberCommandBaseValidator(regionsReadRepository));
             Include(new RequestedByMemberIdValidator(membersReadRepository));
-            RuleFor(c => c.UserId)
-                .NotEmpty();
+            RuleFor(c => c.UserRef)
+                .NotEmpty()
+                .MustAsync(async (command, _, _) => await CheckUserRefExists(command.UserRef))
+                .WithMessage(EmployerPairAlreadyCreatedErrorMessage);
 
             RuleFor(c => c.Organisation)
                 .NotEmpty()
                 .MaximumLength(250);
 
             RuleFor(c => c.AccountId)
-                .NotEmpty()
-                .MustAsync(async (command, _, _) => await CheckAccountIdAndUserIdPair(command.AccountId, command.UserId))
-                .WithMessage(EmployerPairAlreadyCreatedErrorMessage);
-
-            RuleFor(c => c.UserId)
-                .NotEmpty()
-                .MustAsync(async (command, _, _) => await CheckAccountIdAndUserIdPair(command.AccountId, command.UserId))
-                .WithMessage(EmployerPairAlreadyCreatedErrorMessage);
+                .NotEmpty();
         }
 
-        private async Task<bool> CheckAccountIdAndUserIdPair(long accountId, long userId)
+        private async Task<bool> CheckUserRefExists(Guid userRef)
```

Comment:
`CheckUserRefExists` = true could mean that it exists, however true means it does not. So I suggest the method be renamed in line with `IsNewUser`.

Reply:
Method name updated